.github/workflows/test-poetry-build.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           python = "3.10.x"
 
           [tool.poetry.group.dev.dependencies]
-          skypilot = {git = "https://github.com/skypilot-org/skypilot.git", branch = "${{ github.head_ref }}"}
+          skypilot = {git = "${{ github.event.pull_request.head.repo.html_url }}", branch = "${{ github.head_ref }}"}
```

Comment:
I'm not very familiar with gh actions, but looks like {{ github.event.pull_request.head.repo.html_url }} will return something like `https://github.com/skypilot-org/skypilot`. Would we need  add a `.git` at the end? I.e., something like:

```
skypilot = {git = "${{ github.event.pull_request.head.repo.html_url }}.git", branch = "${{ github.head_ref }}"}
```

Reply:
Good point! Added `.git`. Thanks!